http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -441,9 +452,12 @@ protected AuthChallenge resolveCode(String code) {
 
         try {
             TokenValidator tokenValidator = TokenValidator.builder(deployment).build();
-            TokenValidator.VerifiedTokens verifiedTokens = tokenValidator.parseAndVerifyToken(idTokenString, tokenString);
+            String sessionRandValueStr =  getCookieValue(SESSION_RANDOM_VALUE);
```

Comment:
I think this line can be removed now, I don't see sessionRanomValueStr being used.

Reply:
done